commit e7270b0c819732b4ba5f0758f589ad49dbaecb74
Author: Savita Pandit <savita.pandit@PNQ-SPANDIT.local>
Date:   Wed Sep 12 20:03:51 2018 +0530

    changes for PREAPPS-668

diff --git a/src/schema/generated-schema-types.ts b/src/schema/generated-schema-types.ts
index 6522dba..1224524 100644
--- a/src/schema/generated-schema-types.ts
+++ b/src/schema/generated-schema-types.ts
@@ -1092,6 +1092,7 @@ export interface CalendarItemMessageInput {
 
 export interface CalendarItemInviteInput {
 	components: CalendarItemInviteComponentInput[];
+	uid?: string | null;
 }
 
 export interface CalendarItemInviteComponentInput {
@@ -1112,6 +1113,7 @@ export interface CalendarItemInviteComponentInput {
 	status?: InviteCompletionStatus | null;
 	noBlob?: boolean | null;
 	description?: CalendarItemInviteComponentDescriptionInput[] | null;
+	transp?: string | null;
 }
 
 export interface CalendarItemDateTimeInput {
diff --git a/src/schema/schema.graphql b/src/schema/schema.graphql
index 46a06c2..bfb4db6 100644
--- a/src/schema/schema.graphql
+++ b/src/schema/schema.graphql
@@ -1294,6 +1294,7 @@ input SendMessageInput {
 
 input CalendarItemInviteInput {
 	components: [CalendarItemInviteComponentInput]!
+	uid: String
 }
 
 input MailItemEmailAddressInput {
@@ -1319,7 +1320,8 @@ input CalendarItemInviteComponentInput {
 	percentComplete: String
 	status: InviteCompletionStatus
 	noBlob: Boolean
-	description: [CalendarItemInviteComponentDescriptionInput]
+	description: [CalendarItemInviteComponentDescriptionInput],
+	transp: String
 }
 
 input CalendarItemInviteComponentDescriptionInput {
